src/main/java/network/brightspots/rcv/TallyTransfers.java

```diff
@@ -24,20 +24,40 @@
 
 package network.brightspots.rcv;
 
+import static network.brightspots.rcv.Utils.isNullOrBlank;
+
 import java.math.BigDecimal;
 import java.util.HashMap;
 import java.util.Map;
 
 // TallyTransfers class stores summary info on vote transfers
-// used primary as visualizer input to help build Sankey plots
+// used primarily as visualizer input to help build Sankey plots
 class TallyTransfers {
 
+  static final String EXHAUSTED = "exhausted";
+  static final String RESIDUAL_TARGET = "residual surplus";
+  static final String UNCOUNTED = "uncounted";
+  static final String[] RESERVED_STRINGS = {EXHAUSTED, RESIDUAL_TARGET, UNCOUNTED};
+
   // Map of round number to vote transfers which occurred in that round
   // transfers for a round are a map of SOURCE candidate(s) to one or more TARGET candidates.
   // For each target candidate the map value is total vote values received from that source.
   // For round 1 source candidate is marked "uncounted" since the votes had no prior recipient.
   private final Map<Integer, Map<String, Map<String, BigDecimal>>> tallyTransfers = new HashMap<>();
 
+  static boolean candidateStringIsReserved(String candidateString) {
```

Comment:
This seems more like it's exclusively a validation function at this point; should it live in `ContestConfig`?